torchaudio/datasets/gtzan.py

```diff
@@ -998,12 +998,20 @@ def load_gtzan_item(fileid: str, path: str, ext_audio: str) -> Tuple[Tensor, str
 
 
 class GTZAN(Dataset):
-    """
-    Create a Dataset for GTZAN. Each item is a tuple of the form:
-    waveform, sample_rate, label.
+    """Create a Dataset for GTZAN.
+
+    Note:
+        Please see http://marsyas.info/downloads/datasets.html if you are planning to use
+        this dataset to publish results.
 
-    Please see http://marsyas.info/downloads/datasets.html
-    if you are planning to use this dataset to publish results.
+    Args:
+        root (str): Path to the directory where the dataset is found or downloaded.
+        url (str, optional): The URL to download the dataset from.
+            (default: ``"http://opihi.cs.uvic.ca/sound/genres.tar.gz"``)
+        folder_in_archive (str, optional): The top-level directory of the dataset.
+        download (bool, optional): Download dataset if it is not found at root path. (default: ``False``).
+        subset (str, optional): Subset of the dataset. Either ``"training"``, ``"validation"`` or
```

Comment:
Maybe: "Which subset of the dataset to use"? 

What happens when subset is None? I assume the full dataset is used. Maybe something like, "Which subset of the dataset to use. Either None, training, validation, or testing. If None the entire dataset is used. (default: None)." ?

Reply:
Yes, your assumption is correct. Updated the description.